rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/service/LibraryService.kt

```diff
@@ -40,7 +46,9 @@ class LibraryServiceImpl(
         telegramUserId: String
     ): Mono<LibraryDtoResponse> = libraryRepository
         .saveNewLibrary(nameOfNewLibrary, telegramUserId)
-        .onErrorMap { LibraryWithTheSameNameForUserAlreadyExistException("library $nameOfNewLibrary is created already") }
+        .onErrorMap {
+            LibraryWithTheSameNameForUserAlreadyExistException("library $nameOfNewLibrary is created")
+        }
         .map(Library::toDtoResponse)
```

Comment:
**NIT**
As for me onError***{} methods should be last in chain. So line 52 : `.map(Library::toDtoResponse)` is ignored by me. 
Also here - in case of error .map() won't work anyway so I'd say it should be before onErrorMap()